R/error-checks.R

```diff
@@ -19,7 +19,6 @@
 #' df <- data.frame(x = c(1, 2, 3))
 #' isOfType(df, "data.frame")
 #' @export
-
 isOfType <- function(object, type, nullAllowed = FALSE) {
```

Comment:
As decided during the `DataCombined` PR, no empty line between function and its `roxygen` docs.